unified-runtime/source/loader/layers/sanitizer/asan/asan_interceptor.hpp

```diff
@@ -143,6 +144,10 @@ struct ContextInfo {
   std::vector<ur_device_handle_t> DeviceList;
   std::unordered_map<ur_device_handle_t, AllocInfoList> AllocInfosMap;
 
+  ur_shared_mutex InternalQueueMapMutex;
+  std::unordered_map<ur_device_handle_t, std::optional<ManagedQueue>>
```

Comment:
why need "optional"?

Reply:
ManagedQueue doesn't have a default constructor, that will cause build fail if directly use ManagedQueue as map value. And we also don't have a plan to add it, so just wrap it with optional.